source/System.h

```diff
@@ -105,6 +105,11 @@ class System {
 	double HabitableZone() const;
 	// Get the radius of the asteroid belt.
 	double AsteroidBelt() const;
+	// Get the number of ____ in this system
```

Comment:
Full stop, s'il vous plait! And I'd prefer "the named object" to "_____", myself.